app/livechat/server/hooks/processRoomAbandonment.js

```diff
@@ -48,6 +50,8 @@ const getSecondsSinceLastAgentResponse = (room, agentLastMessage) => {
 };
 
 callbacks.add('livechat.closeRoom', (room) => {
+	Apps.getBridges().getListenerBridge().livechatEvent(AppInterface.ILivechatRoomClosedHandler, room);
```

Comment:
What about calling the `livechatEvent` inside the `closeRoom` method [here](https://github.com/RocketChat/Rocket.Chat/blob/develop/app/livechat/server/lib/Livechat.js#L298)?

I think it's a better idea because that method will always be called, what do you think?